samples/highcharts/accessibility/accessible-pattern-fills/demo.js

```diff
@@ -0,0 +1,317 @@
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 3
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 4
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 5
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 6
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 7
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 8
+            }
+        }, {
+            y: 1,
+            color: {
+                patternIndex: 9
+            }
+        }]
+    }]
+});
+
+function getColorPattern(i) {
+    var colors = Highcharts.getOptions().colors,
+        patternColors = [colors[2], colors[0], colors[3], colors[1], colors[4]],
+        patterns = [
+            'M 0 0 L 5 5 M 4.5 -0.5 L 5.5 0.5 M -0.5 4.5 L 0.5 5.5',
+            'M 0 5 L 5 0 M -0.5 0.5 L 0.5 -0.5 M 4.5 5.5 L 5.5 4.5',
+            'M 1.5 0 L 1.5 5 M 4 0 L 4 5',
+            'M 0 1.5 L 5 1.5 M 0 4 L 5 4',
+            'M 0 1.5 L 2.5 1.5 L 2.5 0 M 2.5 5 L 2.5 3.5 L 5 3.5'
+        ];
+
+    return {
+        pattern: {
+            path: patterns[i],
+            color: patternColors[i],
+            width: 5,
+            height: 5
+        }
+    };
+}
+
+
+Highcharts.chart('container-pie', {
+    chart: {
+        type: 'pie'
+    },
+
+    title: {
+        text: ''
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    lang: {
+        accessibility: {
+            chartContainerLabel: 'Pie chart with patterns.'
+        }
+    },
+
+    accessibility: {
+        screenReaderSection: {
+            beforeChartFormat: '<p>Pie chart showing use of patterns.</p>'
+        },
+        landmarkVerbosity: 'one'
+    },
+
+    tooltip: {
+        valueSuffix: '%',
+        borderColor: '#8ae'
+    },
+
+    plotOptions: {
+        series: {
+            dataLabels: {
+                enabled: false
+            }
+        }
+    },
+
+    series: [{
+        data: [{
+            y: 40.6,
+            color: getColorPattern(0)
+        }, {
+            y: 40.1,
+            color: getColorPattern(1)
+        }, {
+            y: 12.9,
+            color: getColorPattern(2)
+        }, {
+            y: 2,
+            color: getColorPattern(3)
+        }, {
+            y: 4.4,
+            color: getColorPattern(4)
+        }]
+    }]
+});
+
+Highcharts.chart('container-line', {
+    chart: {
+        type: 'spline',
+        style: {
+            fontFamily: 'Roboto,Arial'
+        }
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    lang: {
+        accessibility: {
+            chartContainerLabel: 'Line chart with patterns.'
+        }
+    },
+
+    accessibility: {
+        screenReaderSection: {
+            beforeChartFormat: '<p>Line chart showing use of patterns.</p>'
```

Comment:
Actually this one is using dash styles and not patterns - not sure if we need to make that distinction? Thoughts?